docs/docs/guides/labs/dg/incrementally-adopting-dg/migrating-project.md

```diff
@@ -13,69 +13,93 @@ This guide is only relevant if you are starting from an _existing_ Dagster proje
 
 :::
 
-Let's begin with a Dagster project that has some assets:
+Let's begin with a basic existing Dagster project. Our current working
+directory is the project root. Our project defines a Python
+package with a `setup.py` and a single Dagster asset. The asset is exposed in
+a top-level `Definitions` object in `my_existing_project/definitions.py`.
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/1-tree.txt"  />
 
 ## Install dependencies
 
 ### Install the `dg` command line tool
 
-We'll need to install the `dg` command line tool, which is used to scaffold components. We recommend installing `dg` globally using the [`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager; it can also be installed using `pip`.
+We'll need to install the `dg` command line tool. We recommend installing `dg` globally using the [`uv`](https://docs.astral.sh/uv/getting-started/installation/) package manager; it can also be installed using `pip`.
 
 <CliInvocationExample contents="uv tool install dagster-dg" />
 
 ### Install `dagster-components`
 
-Next, we'll need to install the `dagster-components` package.
+Next, we'll need to add `dagster-components` as a dependency of our project. Add it to `install_requires` in `setup.py`:
 
-Though this is optional, we generally recommend using a separate virtual environment for each project, which can be accomplished using `uv`:
+<CodeExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/2-setup.py" language="python" title="setup.py" />
 
-<Tabs>
-    <TabItem value='before' label='Using uv virtual environment'>
-        <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/3-uv-venv.txt" />
+If you have an existing Python virtual environment, you can simply reinstall
+your project into that environment. We don't have one yet, so we'll create one
+and activate it:
 
-        Then, we can use `uv sync` to install the dependencies from our `pyproject.toml`, and then install the `dagster-components` package:
-        <CliInvocationExample path="docs_snippets/docs_snippets/guides/dg/migrating-project/4-uv-freeze.txt" />
-    </TabItem>
-    <TabItem value='after' label='Using pip'>
-        <CliInvocationExample contents="pip install dagster-components" />
-    </TabItem>
-</Tabs>
+<CliInvocationExample contents="python -m venv .venv && source .venv/bin/activate" />
+
+Now we install (or reinstall) our project into the venv:
+
+<CliInvocationExample contents="pip install -e ." />
 
 ## Update project structure
 
-### Update `pyproject.toml`
+### Add `pyproject.toml`
+
+To get `dg` to recognize our project, we need to have a `pyproject.toml` file
+with a `tool.dg` section. Projects that are already using `pyproject.toml`
```

Comment:
Explictly say if you are _are_ using pyproject.toml, you need to add it, but that it serves purely as a repository of config.